source/ui/PGM/PGMPersistentView.cpp

```diff
@@ -0,0 +1,220 @@
+#include "PGMPersistentView.h"
+#include "beamline/PGM/PGMBeamline.h"
+
+#include <QGridLayout>
+#include <QLabel>
+#include <QGroupBox>
+
+PGMPersistentView::PGMPersistentView(QWidget *parent) :
+    QWidget(parent)
+
+{
```

Comment:
Might be an idea to have all these controls in a PGMBeamStatusView, and then have the persistent view contain one of those. That will help with the potential for the persistent view code from becoming fairly cluttered in the future.